pac4j-core/src/main/java/org/pac4j/core/http/HttpActionAdapter.java

```diff
@@ -0,0 +1,36 @@
+/*
+  Copyright 2012 - 2015 pac4j organization
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+package org.pac4j.core.http;
+
+import org.pac4j.core.context.WebContext;
+
+/**
+ * HTTP action adapter.
+ *
+ * @author Jerome Leleu
+ * @since 1.8.2
+ */
+public interface HttpActionAdapter {
```

Comment:
In Play, see: https://github.com/pac4j/play-pac4j/pull/89/files#diff-0f312e3d8b53e5fb5a98dd7a144a7c2eR34 and with SparkJava as well. Though, the upgrade on Spark is in progress and I will only submit a PR in a few days...